src/blockchain/shelley/utils.js

```diff
@@ -1,8 +1,8 @@
-// flow
+// @flow
 
-import { TxInputType } from '../common'
+import { CERT_TYPE } from './certificate'
 
-const fragmentToObj = (fragment: any, extraData: {}): TxType => {
+const fragmentToObj = (fragment: any, extraData: {}) => {
```

Comment:
Why is the return type removed here?

Reply:
There is no such time in codebase.I suggest there should be something like `ShelleyTxType`

Reply:
ah, ok makes sense for now